packages/components/src/utils/math.js

```diff
@@ -71,28 +71,29 @@ export function clamp( value, min, max ) {
 }
 
 /**
- * Clamps a value based on a min/max range with rounding
+ * Rounds a value to the nearest step offset by a minimum.
  *
- * @param {number | string} value The value.
- * @param {number}          min   The minimum range.
- * @param {number}          max   The maximum range.
- * @param {number}          step  A multiplier for the value.
+ * @param {number|string} value The value.
+ * @param {number}        min   The minimum range.
+ * @param {number}        step  A multiplier for the value.
  *
- * @return {number} The rounded and clamped value.
+ * @return {number} The value as a valid step.
  */
-export function roundClamp(
-	value = 0,
-	min = Infinity,
-	max = Infinity,
-	step = 1
-) {
+export function ensureValidStep( value = 0, min = Infinity, step = 1 ) {
```

Comment:
Just noticed that `min = Infinity` seems to make no sense and I left that unchanged. Given that the proper stepping behavior requires starting from `min` a real number has to be used. This is currently accounted for on L86 but this looks like it can be simplified by just defaulting to begin with, i.e. `min = 0`.

Reply:
Actually, I decided none of the defaults are sensible. I removed them in 80ae1db290487ac9eadce24626bec537a905235f. There’s only a single consumer of this and it’s specifying all the arguments.

Maybe this shouldn’t even live in the component package’s top level utils folder. I guess it’s here because `roundClamp` was also used in `FocalPointPicker`. Happy to move it somewhere into number-control if anyone feels it’s a good idea.